examples/lfric/eg3/w3_solver_kernel_mod.F90

```diff
@@ -75,7 +75,7 @@ module w3_solver_kernel_mod
        func_type(Wchi, GH_DIFF_BASIS)        &
        /)
   integer :: gh_shape = GH_QUADRATURE_XYoZ
-  integer :: iterates_over = CELLS
+  integer :: operates_on = CELL_COLUMN
```

Comment:
Is the name changed from `CELL` to `CELL_COLUMN` in the `use argument_mod` statement? It doesn't seem so.

Reply:
Done.